private_gpt/server/ingest/ingest_service.py

```diff
@@ -6,14 +6,17 @@
 from injector import inject, singleton
 from llama_index import (
     Document,
+    JSONReader,
```

Comment:
This may require adding an extra dependency to poetry. Did you check?

Reply:
I did by reading the implementation in `llama_index`: it leverage python's implementation of `json`, so no new dependency.